plan-for-extension-exercise/src/planForExtensionExercise/Shape.java

```diff
@@ -0,0 +1,9 @@
+package planForExtensionExercise;
+
+public abstract class Shape {
```

Comment:
Hi Spencer,

I updated my code in that branch and closed the newest pull request, but
how do I "update" my first pull request?

Thanks, Tracy

On Thu, Feb 5, 2015 at 11:48 AM, Spencer Robinson <notifications@github.com>
wrote:

> In plan-for-extension-exercise/src/planForExtensionExercise/Shape.java
> <https://github.com/siostechcorp/exercises/pull/2#discussion_r24177110>:
>
> > @@ -0,0 +1,9 @@
> > +package planForExtensionExercise;
> > +
> > +public abstract class Shape {
>
> Sorry, I wasn't very clear. Please update this pull request to include the
> change, and close the other pull request.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/siostechcorp/exercises/pull/2/files#r24177110>.
>